cypress/e2e/Login.cy.ts

```diff
@@ -1,9 +1,12 @@
-describe('When a user wants to login with email/password', () => {
+describe('When the user login', () => {
   beforeEach(() => {
-    cy.visit('/login');
+    cy.visit('/');
+    cy.get('[data-cy="nav-login"]').should('be.visible');
+    cy.wait(300);
+    cy.get('[data-cy="nav-login"]').click();
   });
 
-  it('Then the list of post should be shown', () => {
+  it('Then the list of post should be shown', async () => {
```

Comment:
No es necesario que sea `async` y ¿Podrias aprovechar para usar baseUrl de la config de cypress?:
`cy.url().should("equal", "http://localhost:3000/");`

Reply:
Claro, voy a remover el async, y uso el baseUrl.

Reply:
Se me olvido mencionar que apra obtener el baseUrle seria asi: `Cypress.config().baseUrl`